File: src/main/java/net/kunmc/lab/randomskin/event/JoinEventListener.java
package net.kunmc.lab.randomskin.event;

import net.kunmc.lab.randomskin.RandomSkin;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class JoinEventListener implements Listener {

    @EventHandler
    public void on(PlayerJoinEvent e){
        Player p = e.getPlayer();
        if(RandomSkin.rm)
    }
}

File: src/main/java/net/kunmc/lab/randomskin/event/JoinEventListener.kt
package net.kunmc.lab.randomskin.event

import org.bukkit.GameMode
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

class JoinEventListener: Listener {

    @EventHandler
    fun on(e: PlayerJoinEvent){
        if(e.player.gameMode == GameMode.CREATIVE){
            return
        }
    }

}
